src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet.mdx

```diff
@@ -269,6 +269,12 @@ First you will want to [invoke your Lambda function](https://docs.aws.amazon.com
 
 Your telemetry will not appear under New Relic Serverless. Instead, you will find your telemetry data under the New Relic OpenTelemetry Nerdlets.
 
+## Distributed Tracing
+
+In some cases you may see fragmented distributed traces within New Relic with this configuration. This occurs when a trace starts or involves a service that is outside the ADOT context. When this happens, trace spans are usually created by AWS X-Ray and ADOT is not aware of them beyond a trace context being passed to it. This most commonly happens with managed AWS services that ADOT may interact, but that can only be traced using AWS X-Ray.
+
+New Relic has an [X-Ray integration](https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-x-ray-monitoring-integration/) to capture these spans created by X-Ray outside of an ADOT context. However, it has not yet been updated to support OpenTelemetry. Once this integration is updated, you should be able to see a full distributed trace within New Relic using OpenTelemetry.
```

Comment:
I don't think we should mention the X-Ray integration at all, unless it's to recommend using it without any OpenTelemetry configuration if the fragmented traces are a real blocker for you.

Reply:
Without that this is pretty much a dead end.

Reply:
I totally agree we should prioritize the solution to forward and view _all_ AWS spans in New Relic. But as a general principle I don't think we should use language in the docs that promises particular implementations.